### PR TITLE
Cleanup libanki Utils file

### PR DESCRIPTION
**Pull request by lukstbit**
## Purpose / Description

A cleanup for the utilities file Utils in libanki. Mostly move methods around as those methods didn't really belong to the libanki package and they also had in general only one usage. Also fixes IDE issues and kotlin cleanups(if the libanki package is to be moved to a module we need to remove kotlin cleanups).
I recommend each commit to be reviewed one by one. 

## How Has This Been Tested?

Ran the tests.

## Checklist

- [x] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x] You have commented your code, particularly in hard-to-understand areas
- [x] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

File: AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt [hunk 1]
```diff
@@ -1316,8 +1316,12 @@ open class CardBrowser :
     @get:VisibleForTesting
     val previewIntent: Intent
         get() = if (isInMultiSelectMode && checkedCardCount() > 1) {
+            val longArrayCopy = LongArray(selectedCardIds.size)
+            for ((index, item) in selectedCardIds.withIndex()) {
+                longArrayCopy[index] = item
+            }
```
Comment (dae) [inline AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt:1322]: I think you could probably just do selectedCardIds.toLongArray().

**Review by dae — APPROVED** [review]
Thanks for these!

File: AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt (hunk 1 above)
Comment (lukstbit) [reply AnkiDroid/src/main/java/com/ichi2/anki/CardBrowser.kt]: Much better, thanks!

**Review by BrayanDSO — APPROVED** [review]
I always like removing code. LGTM

**Pull request merged**